src/framework/components/particle-system/component.js

```diff
@@ -574,10 +574,69 @@ Object.assign(pc, function () {
             }
         },
 
+        _update: function (dt) {
+            var emitter = this.emitter;
+            if (!emitter.meshInstance.visible) return;
+
```

Comment:
If the entity/component is disabled, is `emitter.meshInstance.visible` false anyway and therefore the emitter is not updated?